HISTORY.rst

```diff
@@ -2,6 +2,29 @@
 History
 =======
 
+3.44.0 (2021-09-16)
+-------------------
+
+Changes
+
+- CumulusCI uses package version Ids from 1GP releases wherever available, reducing the need to install 1GP packages in an org to build 2GP dependencies. (#2832)
+- We added ``metadata_package_id`` and ``version_id`` options to allow passing ``MetadataPackage`` (prefix 033) and ``MetadataPackageVersion`` (prefix 04t) IDs to the push upgrade tasks. (#2837)
+- ``cci flow info`` will now output all flow options defined. (#2845)
+- We improved error messages for incorrect data mapping files (#2831).
```

Comment:
This one is more of an issue closed, I'd say

Reply:
I see it as a feature - we didn't fix any actual issue, but improved the UX.

Reply:
the issue was that it used to raise an exception that didn't include enough information to tell where the misconfiguration was. but it's a grey area; this can stay

Reply:
er, we should probably add "in the bulk data tasks" though so that people are less inclined to scratch their heads trying to remember if they're using data mapping files